CardDictionaries/HitEffects.php

```diff
@@ -91,9 +91,16 @@ function EVOHitEffect($cardID)
         $combatChainState[$CCS_GoesWhereAfterLinkResolves] = "BOTDECK";
         break;
       case "EVO236":
-        AddLayer("TRIGGER", $mainPlayer, $cardID);
-        break;
-      case "EVO241":
+        if(IsHeroAttackTarget()) {
+          $deck = new Deck($defPlayer);
+          if($deck->Empty()) { WriteLog("The opponent deck is already... depleted."); break; }
+            $deck->BanishTop(banishedBy:$mainPlayer);
+            AddDecisionQueue("SEARCHCOMBATCHAIN", $mainPlayer, "-");
+            AddDecisionQueue("SETDQCONTEXT", $mainPlayer, "Choose which card to banish");
+            AddDecisionQueue("CHOOSECARDID", $mainPlayer, "<-");
+            AddDecisionQueue("ALREADYDEAD", $mainPlayer, "CC,-," . $mainPlayer, 1);
+          }
+          break;      case "EVO241":
```

Comment:
Minor thing, but it looks like a newline is missing here

Reply:
I added the missing break line